kit/popups/src/Sheets/UISheet/UISheet.tsx

```diff
@@ -144,6 +144,11 @@ function SheetContent({
             return undefined;
         }
 
+        if (!hasOpenAnimation) {
+            animate(true);
```

Comment:
if have no animation... then animate with `true`... 🤔 am I missing smth?

Reply:
It's actually means `open`)